packages/gatsby-recipes/src/graphql-server/index.js

```diff
@@ -14,7 +14,9 @@ exports.startGraphQLServer = async (programPath, forceStart) => {
   let { port } = (await getService(programPath, `recipesgraphqlserver`)) || {}
 
   if (!port || forceStart) {
-    port = await detectPort(4000)
+    // Use 50400 as our port as it's a highly composite number! Meaning it has
```

Comment:
This comment is both meaningless and useless. Is this the quality of code we're checking in?